app.js

```diff
@@ -0,0 +1,86 @@
+// import functions and grab DOM elements
+import { getRandThrow, testIfWin } from '/utils.js';
```

Comment:
I think you are missing the . here in the file path, which is causing your import to fail.